setup.cfg

```diff
@@ -0,0 +1,53 @@
+[metadata]
+name = notebook_shim
+version = attr: notebook_shim._version.__version__
+description = Jupyter Notebook as a Jupyter Server extension.
+long_description = file: README.md
+long_description_content_type = text/markdown
+license_file = LICENSE
+author = Jupyter Development Team
+author_email = jupyter@googlegroups.com
+url = https://jupyter.org
+platforms = Linux, Mac OS X, Windows
+keywords = ipython, jupyter
+classifiers =
+    Intended Audience :: Developers
+    Intended Audience :: System Administrators
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: BSD License
+    Programming Language :: Python
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+
+[options]
+zip_safe = False
+include_package_data = True
+packages = find:
+python_requires = >=3.6
```

Comment:
We could also go for `>=3.7` since JupyterLab 4.0 drop Python 3.6: https://github.com/jupyterlab/jupyterlab/pull/11740

Reply:
Done in https://github.com/jupyterlab/notebook_shim/pull/1/commits/514a1947a4128fc692c655c3d903bd47d2c0ec99